tests/integration/conftest.py

```diff
@@ -105,6 +105,26 @@ def inner(workspace_group_name=None, account_group_name=None):
     return inner
 
 
+@pytest.fixture
+def make_ucx_group_with_diff_members(make_random, make_group, make_acc_group, make_user):
```

Comment:
Don't add a fixture if it's used only once. It makes the code convoluted and harder to maintain. Rewrite by inlining the fixture code into that single test

Reply:
Removed this fixture and rewrote it inline with test.